components/lib/config/PrimeVue.js

```diff
@@ -71,6 +71,8 @@ export const defaultOptions = {
         emptySelectionMessage: 'No selected item',
         emptySearchMessage: 'No results found',
         emptyMessage: 'No available options',
+        passwordHide: 'Hide Password',
+        passwordShow: 'Show Password',
```

Comment:
Move this down into the `aria: {` section these are ARIA labels.

Reply:
Done